staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -354,10 +378,39 @@ func (cfgCtlr *configController) digestConfigObjects(newPLs []*flowcontrol.Prior
 			errs = append(errs, errors.Wrap(err, fmt.Sprintf("failed to set a status.condition for FlowSchema %s", fsu.flowSchema.Name)))
 		}
 	}
+	cfgCtlr.addUpdateResult(currResult)
+
 	if len(errs) == 0 {
-		return nil
+		return suggestedDelay, nil
+	}
+	return suggestedDelay, apierrors.NewAggregate(errs)
+}
+
+// shouldDelayUpdate checks to see if a flowschema has been updated too often and returns true if a delay is needed.
+func (cfgCtlr *configController) shouldDelayUpdate(flowSchemaName string) bool {
+	numUpdatesInPastMinute := 0
+	oneMinuteAgo := time.Now().Add(-1 * time.Minute)
```

Comment:
nit: time.Now().Sub(time.Minute)

Reply:
Time::Sub is typed `Time -> Duration`, not `Duration -> Time`.

Reply:
ah thanks for catching